### PR TITLE
Fix the support power SpeechNotifications not being played to the correct player

### LINKED ISSUE #11976: [RA] "A-bomb launch detected" does not play [playtest-20160904]
Since the new playtest, the "A-bomb launch detected"-sound message does not play anymore.

### PR DESCRIPTION
**Pull request by abcdefg30**
Fixes #11976.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Extracted the duplicated code into a helper method.

**Comment by jaZzKCS** [comment]
Is this one also related? #11976

**Comment by obrakmann** [comment]
:+1:

File: OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs [hunk 1]
```diff
@@ -120,5 +120,17 @@ public virtual void Activate(Actor self, Order order, SupportPowerManager manage
...
 			}
 		}
+
+		public virtual void PlayLaunchSounds()
+		{
+			var renderPlayer = Self.World.RenderPlayer;
+			var isAllied = Self.Owner.IsAlliedWith(renderPlayer);
+			Game.Sound.Play(isAllied ? Info.LaunchSound : Info.IncomingSound);
+
+			// IsAlliedWith returns true if renderPlayer is null, so we are save here.
```
Comment (RoosterDragon) [inline OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs]: `save` -> `safe`

**Comment by RoosterDragon** [comment]
:+1:

File: OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs (hunk 1 above)
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/SupportPowers/SupportPower.cs]: Thanks, fixed.

**Comment by abcdefg30** [comment]
Updated.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/e6c016706ddf659503c0fc377e65242c5ba375a3)

**Pull request merged**